scripts/src/java/org/oppia/android/scripts/proto/coverage.proto

```diff
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package proto;
+
+option java_package = "org.oppia.android.scripts.proto";
+option java_multiple_files = true;
+
+// Coverage Report that contains the bazel coverage data retrieved from the
+// Bazel coverage execution.
+message CoverageReport {
+  // The test target for which the coverage report is generated.
+  string bazel_test_target = 1;
+  // A list of files covered in this coverage report.
+  repeated CoveredFile covered_file = 2;
+}
+
+// Information about the single file that was covered during the tests.
+message CoveredFile {
```

Comment:
After looking at the parsing logic, how many of these properties do we actually need? I highly suggest dropping everything that doesn't actually have use for enforcement or final HTML/MD report generation.

Reply:
Just removed everything else and kept only those related to lines:
```
message CoverageReport {
  string bazel_test_target = 1;
  string file_path = 2;
  string file_sha1_hash = 3;
  repeated CoveredLine covered_line = 4;
  int32 lines_found = 5;
  int32 lines_hit = 6;
}

message CoveredLine {
  int32 line_number = 1;
  Coverage coverage = 2;
}
```